generators/advanced/templates/test/mocha.opts

```diff
@@ -0,0 +1,4 @@
+--recursive
+--require ./test/env.js
+--timeout 20000
+--exit
```

Comment:
mocha.opts is deprecated, so we should start using the new approach: https://mochajs.org/#configuring-mocha-nodejs

Reply:
nice catch, tracked here: #58